packer/templates/windows_2008_r2.json

```diff
@@ -184,6 +183,13 @@
     {
       "type": "windows-restart"
     },
+    {
+      "type": "powershell",
+      "elevated_user": "Administrator",
+      "elevated_password": "vagrant",
+      "scripts": "{{user `scripts_dir`}}/installs/setup_iis.bat",
+      "pause_before": "60s"
```

Comment:
I didn't test it without the pause. This was kind of left over during testing trying to get it to work. Tonight I will make sure to pull a fork from the master and add these changes and try without the pause. This will make everything cleaner also.